engines/sci/detection.cpp

```diff
@@ -119,10 +119,10 @@ static const PlainGameDescriptor s_sciGameTitles[] = {
 	{"hoyle4",          "Hoyle Classic Card Games"},
 	{"inndemo",         "ImagiNation Network (INN) Demo"},
 	{"kq6",             "King's Quest VI: Heir Today, Gone Tomorrow"},
-	{"laurabow2",       "Laura Bow 2: The Dagger of Amon Ra"},
-	{"qfg1vga",         "Quest for Glory I: So You Want to Be a Hero"},	// Note: There was also a SCI0 version of this (further up)
+	{"laurabow2",       "Roberta Williams' Laura Bow in The Dagger of Amon Ra"}, //also called Laura bow II
+	{"qfg1vga",         "Quest for Glory: So You Want to Be a Hero"},	// Note: There was also a SCI0 version of this (further up)
```

Comment:
The remake _was_ titled "Quest for Glory I" https://www.mobygames.com/game/quest-for-glory-i-so-you-want-to-be-a-hero/cover-art/gameCoverId,317718/